test/JDBC/expected/sys-type_info_internal-vu-prepare.out

```diff
@@ -0,0 +1 @@
+GO
```

Comment:
It is okay not having prepare and cleanup files. Let's remove empty files (Ref: https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/693/files#diff-e4eded5d52f6592e8a446b734ba4940fbc47f222910d1b753f2f2897e7ad5106)

Reply:
Now the files are not empty any more. 
Prep and clean up is required now to test the sys.table_types_internal view after removing 
GRANT SELECT ON sys.table_types_internal TO PUBLIC
from the sys_views.sql